cpp/src/utilities/bit_util.cuh

```diff
@@ -17,9 +18,13 @@
  */
 #pragma once
 
+#include <utilities/cudf_utils.h>
+#include <utilities/integer_utils.hpp>
+
 #include <cudf/types.h>
 
-#include <stdint.h>
+#include <cstdint>
+#include <climits>
 #include <string>
 
 #ifndef CUDA_HOST_DEVICE_CALLABLE
```

Comment:
Why not just include `utilities/cudf_utils.h` which defines these same macros? Then we don't repeat ourselves.